### PR TITLE
Batch / Advanced Search doc updates

### PR DESCRIPTION
**Pull request by allanaaa**
For merging with the release of 4.0.

Closes https://github.com/omeka/omeka-s/issues/1909 - items page first.

### CODE REVIEW

File: docs/content/items.md [hunk 1]
```diff
@@ -271,21 +271,31 @@ Batch actions are as follows:
 - Delete selected: delete only the items that are selected on the page
 - Delete all: delete all the items returned by a search (default is all items).
 
-Choose one of these options and then click "Go".
+The number of results per page is set in the [installation's global settings](../admin/settings.md#general).
+
+You can first use the "🔍 Advanced search" link to narrow down the items to a subset for editing. For example, you can use [Advanced search](../search.md#item-advanced-search) to limit to items that are not in any item sets, or items that are owned by a specific user. In the image below, the search is set to capture any item without a "Date created" value.
```
Comment (allanaaa) [inline docs/content/items.md:276]: Line 276: Unicode magnifying glass icon, yes or no? I kinda like it but I haven't totally looked into rendering issues yet.

**Comment by allanaaa** [comment]
Partial fix for #1910 too, since the media-language bit appears in batch editing. I'll add more to close it totally.

**Comment by allanaaa** [comment]
Once this is approved it should get tagged to correspond with whenever the next release happens from the develop branch. I haven't quite figured out how to version these things yet.

**Review by jimsafley — APPROVED** [review]

**Comment by allanaaa** [comment]
Closes https://github.com/omeka/omeka-s/issues/1933 now too.

**Pull request merged**